src/components/markdown-input.js

```diff
@@ -67,6 +71,14 @@ function MarkdownInput({ minRows, disabled, inline, debounced, renderAnchors, va
     onChange(event);
   };
 
+  const handleBlur = event => {
+    const sanitizedValue = gfm.makeCdnResourcesPortable(value, clientConfig.cdnRootUrl);
+    if (value !== sanitizedValue) {
+      onChange({ target: { value: sanitizedValue } });
```

Comment:
If the user of this even will expect an event, they might try to use functionality like `event.preventDefault()` and this would throw.
I think we should return here the real event, with only setting
```
event.target.value = sanitizedValue;
```

Reply:
The real event unfortunately is the wrong one (the `blur` event, while I'm trying to squeeze in a fake `change` event before the blur finishes. I think the "correct" solution would be (what also Antd does for non-DOM-conformant components) to change the argument of the `onChange` callback directly to the value instead of a DOM-like event structure with `target` etc. What do you think (it would required changing quite some usages in all plugins though)?

Reply:
I see, true, it's not the dedicated event handler there but the `onChange` one.

To be honest I don't really like having a component called `Input` using the direct value in a `onChange` callback. I noticed it could lead to problems in the `NeverScrollingTextArea` when I introduced the `debounce` toggle and had to return either the real `TextArea` or the wrapper, and had then a diverging signature for `onChange`.

In this case I would either leave it like this or try to create a [synthetic event](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event) so that we don't end up with unexpected exceptions thrown with the use of the event in the `onChange` callback. Not sure if this option works though 😬

Reply:
That's even better, I like the use of the native event! 😁 👏 Good solution! Hope that fixes..2 birds with one commit 😆